src/stores/A_store.js

```diff
@@ -153,6 +153,45 @@ class A_store extends ReduceStore {
                 }
             }
 
+            case ActionTypes.DSTARLITE: {
+                const {player, end, pathGrid} = state;
+                const {grid} = pathGrid;
+                const DSL = new Dstarlite(grid[player.y][player.x], grid[end.y][end.x], Cell.heuristic);
+                DSL.beginPathfinding();
+
+                return {...state, DSL, stage: 'DSTAR_STEP'};
+            }
+
+            case ActionTypes.DSTAR_STEP: {
+                const {DSL, seenCells, revealedGrid, end} = state;
+                let {pathGrid} = state;
```

Comment:
ideally group `const`s together and then `let`s

Reply:
🐇